tools/runner/logsaver.go

```diff
@@ -1,145 +1,103 @@
+/*
+Copyright 2021 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package runner
 
 import (
 	"bytes"
 	"context"
-	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
-	"strings"
 
 	grpcv1 "github.com/grpc/test-infra/api/v1"
-	"github.com/grpc/test-infra/status"
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	corev1types "k8s.io/client-go/kubernetes/typed/core/v1"
 )
 
-// LogSaver provides functionality to save pod logs to files.
-type LogSaver struct {
-	podsGetter corev1types.PodsGetter
-}
-
-// NewLogSaver creates a new LogSaver object.
-func NewLogSaver(podsGetter corev1types.PodsGetter) *LogSaver {
-	return &LogSaver{
-		podsGetter: podsGetter,
-	}
-}
-
-// SavePodLogs saves pod logs to files with same name as pod.
-// This function returns a map where pods are keys and values are the filepath
-// of the saved log.
-func (ls *LogSaver) SavePodLogs(ctx context.Context, loadTest *grpcv1.LoadTest, podLogDir string) (*SavedLogs, error) {
-	savedLogs := NewSavedLogs()
-
-	// Get pods for this test
-	pods, err := ls.getTestPods(ctx, loadTest)
-	if err != nil {
-		return savedLogs, err
-	}
+// SaveAllLogs saves all logs to files and return their LogInfo.
```

Comment:
The godoc comments don't need an empty line, and don't need to repeat the name of the function. So if the first line is // SaveAllLogs saves..., the second line can just continue and explain: // This function ... We can fix the docstrings once the other issues are addressed.